src/index.html

```diff
@@ -1,12 +1,360 @@
+          </li>
+
+          <li class="header__item">
+            <a 
+            href="index.html" 
+            class="header__item-link" 
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Gadgets </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Tablets </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Photo </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="/images/imac.jpeg"
+            alt="Apple A1419 iMac 27 Retina 5K"
+            width="160"
+            height="134"
+            class="card__image"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__block">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__block card__block--second">
+            <p class="card__price">Price:</p>
+
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a 
+          href="#" 
+          class="card__link" 
+          data-qa="card-hover"> 
+            Buy 
+          </a>
+        </div>
+        
+        <div class="catalog__card card">
+          <img
+            src="/images/imac.jpeg"
+            alt="Apple A1419 iMac 27 Retina 5K"
+            width="160"
+            height="134"
+            class="card__image"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__block">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__block card__block--second">
+            <p class="card__price">Price:</p>
```

Comment:
```suggestion
          <div class="card__block card__block--second">
          
            <p class="card__price">Price:</p>
```  Fix that on all cases